### PR TITLE
Notifications: allow override of title element, as the h5 might disturb the heading hierarchy

### PR DESCRIPTION
**Pull request by edlerd**
## Done

- Make the wrapper tag for Notification title configurable. The default `h5` might disturb the heading hierarchy of the page.

### Storybook

To see rendered examples of all react-components, run:

```shell
yarn start
```

### QA in your project

from `react-components` run:

```shell
yarn build
npm pack
```

Install the resulting tarball in your project with:

```shell
yarn add <path-to-tarball>
```

### QA steps

- check that notifications still render the same as before

### CODE REVIEW

File: src/components/Notification/Notification.tsx [hunk 1]
```diff
@@ -97,6 +101,7 @@ const Notification = ({
   timeout,
   timestamp,
   title,
+  titleElement: Component = "h5",
```
Comment (ClementChaumel) [inline src/components/Notification/Notification.tsx:104]: Here I feel like renaming the prop makes the render bit slightly more confusing

Comment (ClementChaumel) [reply src/components/Notification/Notification.tsx:104]: Ah, right. Got it

Comment (edlerd) [reply src/components/Notification/Notification.tsx]: Renamed this to `TitleComponent` just now.

**Comment by edlerd** [comment]
Answered all the open questions above.
I rebased on main and the cypress tests started to fail. Seems like Tooltip tests in main are broken atm.

**Comment by ClementChaumel** [comment]
> Answered all the open questions above. I rebased on main and the cypress tests started to fail. Seems like Tooltip tests in main are broken atm.

Thanks! I'll look at fixing the issue with tooltips then we can merge this!

**Review by ClementChaumel — APPROVED** [review]
Nice work 👌️

**Pull request merged**